src/NaturalLanguage/Annotation.php

```diff
@@ -0,0 +1,263 @@
+ *         echo $entity['type'];
+ *     }
+ *     ```
+ *
+ *     @see https://cloud.google.com/natural-language/reference/rest/v1beta1/Entity Entity type documentation
+ *
+ *     @return array|null
+ * }
+ * @method language() {
+ *     Returns the language of the document.
+ *
+ *     Example:
+ *     ```
+ *     echo $annotation->language();
+ *     ```
+ *
+ *     @return string
+ * }
+ */
+class Annotation
+{
+    use CallTrait;
+
+    /**
+     * @var array The annotation's metadata.
+     */
+    private $info;
+
+    /**
+     * Create an annotation.
+     *
+     * This class is created internally by
+     * {@see Google\Cloud\NaturalLanguage\NaturalLanguageClient} and is used to
+     * represent various document analyzation results.
+     *
+     * This class should not be instantiated externally.
+     *
+     * Annotations are returned by
+     * {@see Google\Cloud\NaturalLanguage\NaturalLanguageClient::analyzeEntities()},
+     * {@see Google\Cloud\NaturalLanguage\NaturalLanguageClient::analyzeSentiment()},
+     * {@see Google\Cloud\NaturalLanguage\NaturalLanguageClient::analyzeSyntax()} and
+     * {@see Google\Cloud\NaturalLanguage\NaturalLanguageClient::annotateText()}.
+     *
+     * @param array $info The annotation's metadata.
+     */
+    public function __construct(array $info = [])
+    {
+        $this->info = $info;
+    }
+
+    /**
+     * Returns the full response from the API.
+     *
+     * Example:
+     * ```
+     * $info = $annotation->info();
+     * ```
+     *
+     * @codingStandardsIgnoreStart
+     * @see https://cloud.google.com/natural-language/reference/rest/v1beta1/documents/annotateText#response-body Annotate Text documentation
+     * @codingStandardsIgnoreEnd
+     *
+     * @return array
+     */
+    public function info()
+    {
+        return $this->info;
+    }
+
+    /**
+     * Returns the sentiment of the document.
+     *
+     * Example:
+     * ```
+     * $sentiment = $annotation->sentiment();
+     *
+     * if ($sentiment['polarity'] > 0) {
+     *    echo 'This is a positive message.';
+     * }
+     * ```
+     *
+     * @see https://cloud.google.com/natural-language/reference/rest/v1beta1/Sentiment Sentiment type documentation
+     *
+     * @return array|null
+     */
+    public function sentiment()
+    {
+        return isset($this->info['documentSentiment']) ? $this->info['documentSentiment'] : null;
+    }
+
+    /**
+     * Returns an array of tokens filtered by the given tag.
+     *
+     * Example:
+     * ```
+     * $tokens = $annotation->fetchTokensByTag('NOUN');
+     *
+     * foreach ($tokens as $token) {
+     *     echo $token['lemma'];
+     * }
+     * ```
+     *
+     * @codingStandardsIgnoreStart
+     * @see https://cloud.google.com/natural-language/reference/rest/v1beta1/documents/annotateText#tag Token tags documentation
+     * @codingStandardsIgnoreEnd
+     *
+     * @return array|null
+     */
+    public function fetchTokensByTag($tag)
```

Comment:
Or maybe `findTokensByX` if you prefer, though I like the more concise name better.